README.md

```diff
@@ -1,6 +1,6 @@
 # etcd operator
 
-[![Build Status](https://jenkins-etcd-public.prod.coreos.systems/buildStatus/icon?job=etcd-operator-master)](https://jenkins-etcd-public.prod.coreos.systems/job/etcd-operator-master/)
+[![Build Status](https://jenkins-etcd-public.prod.coreos.systems/buildStatus/icon?job=coreos-etcd-operator-master)](https://jenkins-etcd-public.prod.coreos.systems/job/coreos-etcd-operator-master/)
```

Comment:
2. I misunderstood at first and had changed the job name as well. It's back to etcd-operator-master-k8s-master.